src/lib/ai/services/__tests__/PersonalizationService.test.ts

```
@@ -120,7 +120,7 @@ describe('PersonalizationService', () => {
   } as TherapeuticTechnique
 
   const sampleProfile: Partial<ClientPreferenceProfile> = {
-    clientId: 'example-client-id',
+    clientId: testId || 'example-client-id',
```

ESLint

[error] 'testId' is not defined.

(no-undef)